monai/apps/pathology/datasets.py

```diff
@@ -52,8 +53,8 @@ def __init__(
         region_size: Union[int, Tuple[int, int]],
         grid_shape: Union[int, Tuple[int, int]],
         patch_size: int,
```

Comment:
the typing is fine but I will homogenize it across other classes as well to be `(height, width)` of the patch. It is better to merge #1876 first to avoid conflicts.

Reply:
please address it in this PR or #1876, I'll look into both PRs